enhancements/console/dynamic-plugins.md

```diff
@@ -272,6 +274,21 @@ available and the user must refresh their browser to see the changes. (We won't
 refresh the page automatically to avoid possibly losing data if the user is
 entering something into a form or the YAML editor.)
 
+### Localization
+
+Dynamic plugins will need to contain their own localization files and serve them
```

Comment:
I'd preference this by saying that console uses react-i18next for i18n, and plugins must use react-i18next as well.